src/Tooltip/examples/positions.js

```diff
@@ -0,0 +1,44 @@
+class Example extends React.Component {
```

Comment:
@FrancescoCioria this example has a lot of layout-related noise... Not sure if we want to keep it. What do you think?